src/formatter.js

```diff
@@ -17,10 +17,10 @@ class XunitFormatter extends JsonFormatter {
                     }
                 })
             }
-            log(new xml2js.Builder().buildObject(xml));
+            log(new xml2js.Builder({
+                xmldec: { noValidation: true }
```

Comment:
can you describe please why do you decide to add `boValidation` here?

Reply:
it fails in case if non ascii chars present in error message (e.g it comes from playwright errors), however it's not a problem for xunit reader

Reply:
got it, thanks for clarification